specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.2-preview.2/TextAnalytics.json

```diff
@@ -0,0 +1,3477 @@
+        "message"
+      ],
+      "properties": {
+        "code": {
+          "type": "string",
+          "enum": [
+            "LongWordsInDocument",
+            "DocumentTruncated"
+          ],
+          "x-ms-enum": {
+            "name": "WarningCodeValue",
+            "modelAsString": true
+          },
+          "description": "Error code."
+        },
+        "message": {
+          "type": "string",
+          "description": "Warning message."
+        },
+        "targetRef": {
+          "type": "string",
+          "description": "A JSON pointer reference indicating the target object."
+        }
+      }
+    },
+    "ExtractiveSummarizationTask": {
+      "type": "object",
+      "properties": {
+        "parameters": {
+          "type": "object",
+          "properties": {
+            "model-version": {
+              "default": "latest",
+              "type": "string"
+            },
+            "loggingOptOut": {
+              "default": false,
+              "type": "boolean"
+            },
+            "stringIndexType": {
+              "$ref": "#/definitions/StringIndexType"
+            },
+            "sentenceCount": {
+              "default": 3,
+              "type": "integer"
+            },
+            "sortBy": {
+              "default": "Offset",
+              "enum": [
+                "Offset",
+                "Rank"
+              ],
+              "type": "string"
+            }
+          }
+        },
+        "taskName": {
+          "type": "string"
+        }
+      }
+    },
+    "CustomEntitiesTask": {
+      "properties": {
+        "parameters": {
+          "properties": {
+            "project-name": {
+              "type": "string"
+            },
+            "deployment-name": {
+              "type": "string"
+            },
+            "loggingOptOut": {
+              "default": false,
+              "type": "boolean"
+            },
+            "stringIndexType": {
+              "$ref": "#/definitions/StringIndexType"
+            }
+          },
+          "type": "object",
+          "required": [
+            "project-name",
+            "deployment-name"
+          ]
+        },
+        "taskName": {
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "CustomSingleClassificationTask": {
+      "properties": {
+        "parameters": {
+          "properties": {
+            "project-name": {
+              "type": "string"
+            },
+            "deployment-name": {
+              "type": "string"
+            },
+            "loggingOptOut": {
+              "default": false,
+              "type": "boolean"
+            }
+          },
+          "type": "object",
+          "required": [
+            "project-name",
+            "deployment-name"
+          ]
+        },
+        "taskName": {
+          "type": "string"
+        }
+      },
+      "type": "object"
+    },
+    "CustomMultiClassificationTask": {
```

Comment:
The definition for this is basically the same as the CustomClassificationTask above.  Do you think there may be divergence of these two task types in the future in terms of parameters, or could we theoretically use the same definition for both?